src/components/Article.jsx

```diff
@@ -1 +1,46 @@
-// export const Article = () => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = (props) => {
+  const {
+    title,
+    date,
+    text,
+  } = props;
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <div className="box">
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </div>
+  );
+};
+
+Article.defaultProps = {
+  title: 'title',
+  date: new Date(),
+  text: 'text',
+};
+
+Article.propTypes = {
+  title: PropTypes.string,
+  date: PropTypes.func,
```

Comment:
Also, I'm not sure, but func maybe means that you can't be sure that this value will be related to date, I found in the Net, that we can use this to check for date ```PropTypes.instanceOf(Date)```